geniza/corpus/templates/corpus/snippets/document_result.html

```diff
@@ -55,11 +55,11 @@ <h2 class="title">
                     {# use first language code, if any; otherwise empty lang attribute to indicate unknown language #}
                     {# NOTE: this is problematic for documents with multiple primary languages #}
                     {% if document_highlights.transcription %}
-                        <div class="transcription" lang="{{ lang }}">{% for snippet in document_highlights.transcription %}{{ snippet.strip|safe|linebreaks }}{% endfor %}</div>
+                        <div class="transcription" lang="{{ lang }}">{% for snippet in document_highlights.transcription %}{{ snippet.strip|safe }}{% endfor %}</div>
```

Comment:
@rlskoeser I'm running into issues here when searching on transcription text; the highlighted snippet seems to be closing `li` tags that it hasn't opened, which breaks the search results `ol`. At least, that's my best guess at what's happening; my search term was "הן" if you want to try to replicate.

Reply:
ooh, you know I wondered if that might come up but didn't test thoroughly enough to run into it...  let me look into options for dealing with it